### PR TITLE
Us01 home page

### PR DESCRIPTION
**Pull request by Salma-D38**

### CODE REVIEW

**Review by MaxAnxxi — APPROVED** [review]
Reviewed done

File: client/src/pages/HomePage/HomePageStyle.module.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  gap: 10rem;
+}
+
+.image {
+  height: 20rem;
+  width: 20rem;
+}
+
+.button {
+  padding-left: 2rem;
```
Comment (HazeFury) [inline client/src/pages/HomePage/HomePageStyle.module.css]: 4 propriétés de padding alors que tu pourrais en mettre une seule.

et arrête l'escroquerie 😠

**Review by HazeFury — CHANGES_REQUESTED** [review]
PPPPPPPPPPPPPPPPPPPPPPPPPPPAAAAAAAAAAAAAAAAAAAAAAAAAADDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDIIIIIIIIIIIIIIIIIIIIIIIIIIIINNNNNNNNNNNNNNNNNNNGGGGGGGGGGGGGGGGGGGGGGGGGGG

File: client/src/pages/HomePage/HomePageStyle.module.css (hunk 1 above)
Comment (MaxAnxxi) [reply client/src/pages/HomePage/HomePageStyle.module.css]: Ouais t'abuses vraiment !

Comment (Salma-D38) [reply client/src/pages/HomePage/HomePageStyle.module.css]: PPPPPPPPPPPPPPPPPPPPPPPPPPPAAAAAAAAAAAAAAAAAAAAAAAAAADDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDIIIIIIIIIIIIIIIIIIIIIIIIIIIINNNNNNNNNNNNNNNNNNNGGGGGGGGGGGGGGGGGGGGGGGGGGG CCCCCCCCCCCCCCOOOOOOOOOOOOOOOORRRRRRRRRRRRRRRRIIIIIIIIIIIIIIIIIIIIIIIGGGGGGGGGGGGGGGGEEEEEEEEEEEEEEEEEEEEEEEEEEEEE

**Review by HazeFury — APPROVED** [review]
Bravo 👍 💯 🥇

**Pull request merged**